src/feeds/ChainlinkCurveAssetFeed.sol

```diff
@@ -4,35 +4,25 @@ import {ICurvePool} from "src/interfaces/ICurvePool.sol";
 import {IChainlinkBasePriceFeed} from "src/interfaces/IChainlinkFeed.sol";
 
 // Combined Chainlink and Curve price_oracle, allows for additional fallbacks to be set via ChainlinkBasePriceFeed
-
-contract ChainlinkPriceOracleStablePriceFeed {
+/// @dev This implementation is for Curve pools when Target is the coin at index 0 in Curve pool `coins` array
```

Comment:
Can we always assume that the target will be index 0 or should we generalize it to take coin index parameters?